### PR TITLE
(PC-13043) refactor(icons): replace all GenericInfoPage icons 

### PR DESCRIPTION
**Pull request by yorickeando**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-13043

## Checklist

I have:

- [x] Made sure the title of my PR follows the [convention](1) `($jira) $type($scope): $summary`.
- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [ ] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** or **github dev name** for any added TODO / FIXME.
- [ ] Added new reusable components to **AppComponents** page and communicate to the team on slack.
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion](2)
- [ ] Made sure translations file is up to date (`yarn translations:extract`)

## Screenshots (quelques exemples)

Capture d’écran 2022-01-25 à 15 45 49
Capture d’écran 2022-01-25 à 14 55 11

Capture d’écran 2022-01-25 à 15 51 04
Capture d’écran 2022-01-25 à 15 51 35
Capture d’écran 2022-01-25 à 15 51 56
Capture d’écran 2022-01-25 à 15 52 16

[1]: https://github.com/pass-culture/pass-culture-app-native/blob/master/doc/standards/pr-title.md
[2]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/features/auth/components/QuitSignupModal.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import { ButtonTertiaryWhite } from 'ui/components/buttons/ButtonTertiaryWhite'
 import { GenericInfoPage } from 'ui/components/GenericInfoPage'
 import { AppFullPageModal } from 'ui/components/modals/AppFullPageModal'
 import { Spacer } from 'ui/components/spacer/Spacer'
-import { Error } from 'ui/svg/icons/Error'
+import { ErrorIllustration } from 'ui/svg/icons/ErrorIllustration'
```
Comment (antoinewg) [inline src/features/auth/components/QuitSignupModal.tsx:13]: pk pas 'ui/svg/illustrations/Error' ?

File: src/features/cheatcodes/pages/AppComponents/Illustrations.tsx [hunk 2]
```diff
@@ -10,38 +10,29 @@ import { BirthdayCake } from 'ui/svg/icons/BirthdayCake'
...
 import { SadFaceDeprecated } from 'ui/svg/icons/SadFace_deprecated'
 import { Star } from 'ui/svg/icons/Star'
-import { StarDeprecated } from 'ui/svg/icons/Star_deprecated'
 import { TicketBooked } from 'ui/svg/icons/TicketBooked'
-import { TicketBookedDeprecated } from 'ui/svg/icons/TicketBooked_deprecated'
 import { IconInterface } from 'ui/svg/icons/types'
 import { UserBlocked } from 'ui/svg/icons/UserBlocked'
-import { UserBlockedDeprecated } from 'ui/svg/icons/UserBlocked_deprecated'
 import { UserError } from 'ui/svg/icons/UserError'
-import { UserErrorDeprecated } from 'ui/svg/icons/UserError_deprecated'
```
Comment (antoinewg) [inline src/features/cheatcodes/pages/AppComponents/Illustrations.tsx:44]: :tada:

**Review by antoinewg — APPROVED** [review]
okey sinon 👍

File: src/features/auth/components/QuitSignupModal.tsx (hunk 1 above)
Comment (yorickeando) [reply src/features/auth/components/QuitSignupModal.tsx:13]: On pourrait faire un dossier illustrations c'est vrai, ça se défend. En tout cas pas dans cette PR haha

**Pull request merged**